There's a typescript error:

`yarn run tsc --build packages/data-stores/tsconfig.json`

```
packages/data-stores/src/onboard/reducer.ts:400:7 - error TS2322: Type '(state: string | undefined, action: OnboardAction) => string | false' is not assignable to type 'Reducer<string, OnboardAction>'.
  Type 'string | false' is not assignable to type 'string'.
    Type 'boolean' is not assignable to type 'string'.

400 const ecommerceFlowRecurType: Reducer< string, OnboardAction > = ( state = '', action ) => {
          ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

--- a/packages/data-stores/src/onboard/reducer.ts
+++ b/packages/data-stores/src/onboard/reducer.ts
@@ -402,7 +402,7 @@ const ecommerceFlowRecurType: Reducer< string, OnboardAction > = ( state = '', a
 		return action.ecommerceFlowRecurType;
 	}
 	if ( action.type === 'RESET_ONBOARD_STORE' ) {
-		return false;
+		return '';
 	}
 	return state;
 };

